### PR TITLE
Fix #1504 -  describe TTL as hops not seconds

### LINKED ISSUE #1504: The Test-Connection help topic mischaracterizes the -TimeToLive parameter
* `-TimeToLive`, despite what the [current documentation](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Test-Connection.md) says, is **NOT a _timeout_ in _seconds_, but the _maximum count of hops_** a packet can make before it is considered expired.

  * As such, it is analogous to `ping.exe`'s `-i <TTL>` parameter.

  * `Test-Connection`'s default value for `-TimeToLive` - `80` - provides a clue that it is _not_ a timeout in _seconds_, as that would be excessively high.

  * As an aside: `Test-Connection` has NO equivalent to `ping.exe`'s truly time-based `-w <ms>` timeout parameter.

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2ad236c](https://github.com/sdwheeler/PowerShell-Docs/pull/3513/commits/2ad236cf42008043011017b669572ade71ac0533)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2ad236c](https://github.com/sdwheeler/PowerShell-Docs/pull/3513/commits/2ad236cf42008043011017b669572ade71ac0533)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 21 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md [hunk 1]
```diff
@@ -15,116 +15,152 @@ Sends ICMP echo request packets ("pings") to one or more computers.
...
+  [-Count <int>] [-Impersonation <ImpersonationLevel>] [-TimeToLive <int>] [-Delay <int>] [-Quiet]
+  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The Test-Connection cmdlet sends Internet Control Message Protocol (ICMP) echo request packets ("pings") to one or more remote computers and returns the echo response replies.
-You can use this cmdlet to determine whether a particular computer can be contacted across an Internet Protocol (IP) network.
 
-You can use the parameters of Test-Connection to specify both the sending and receiving computers, to run the command as a background job, to set a timeout and number of pings, and to configure the connection and authentication.
+The **Test-Connection** cmdlet sends Internet Control Message Protocol (ICMP) echo request packets,
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md]: This and the next Test-Connection are in bold, then the third is code-fenced.  The contrib guide doesn't say what it should be after the first instance... but it should be consistent.

File: reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md [hunk 2]
```diff
@@ -15,116 +15,152 @@ Sends ICMP echo request packets ("pings") to one or more computers.
...
+or pings, to one or more remote computers and returns the echo response replies. You can use this
+cmdlet to determine whether a particular computer can be contacted across an IP network.
+
+You can use the parameters of **Test-Connection** to specify both the sending and receiving
+computers, to run the command as a background job, to set a time-out and number of pings, and to
+configure the connection and authentication.
+
+Unlike the familiar **ping** command, `Test-Connection` returns a **Win32_PingStatus** object
+that you can investigate in Windows PowerShell. You can use the *Quiet* parameter to force it to
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md]: Quiet should be in bold as a parameter.  Repeat for each doc version.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2ad236c](https://github.com/sdwheeler/PowerShell-Docs/pull/3513/commits/2ad236cf42008043011017b669572ade71ac0533)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-3.0&branch=pr-en-us-3513) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/4.0/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-4.0&branch=pr-en-us-3513) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/5.0/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-5.0&branch=pr-en-us-3513) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/5.1/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-5.1&branch=pr-en-us-3513) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/6/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-6&branch=pr-en-us-3513) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901072347090038-3513%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LlmZh0Fx%2BHlGc1STkpoa9D3e3t%2BeD%2FCSHcLgis12l0A%3D&st=2019-01-07T23%3A55%3A51Z&se=2019-02-08T00%3A00%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Management/Test-Connection.md [hunk 3]
```diff
@@ -15,151 +15,162 @@ Sends ICMP echo request packets ("pings") to one or more computers.
...
-PS C:\> Test-Connection "Server01"
 
-Source        Destination     IPV4Address     IPV6Address  Bytes    Time(ms)
-------        -----------     -----------     -----------  -----    --------
-ADMIN1        Server01         10.59.137.44                32       0
-ADMIN1        Server01         10.59.137.44                32       0
-ADMIN1        Server01         10.59.137.44                32       0
-ADMIN1        Server01         10.59.137.44                32       1
+```Output
+Pinging Server01 [172.16.0.55] with 32 bytes of data:
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Management/Test-Connection.md]: All other examples for Server01 use IP 10.59.137.44. Should we keep it consistent for any doing comparisons?

Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Management/Test-Connection.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md (hunk 2 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md]: Fixed

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[1188f78](https://github.com/sdwheeler/PowerShell-Docs/pull/3513/commits/1188f78112ba8ccd09dea67f8d4f23d1b7ae6a7e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[1188f78](https://github.com/sdwheeler/PowerShell-Docs/pull/3513/commits/1188f78112ba8ccd09dea67f8d4f23d1b7ae6a7e)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 20 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[1188f78](https://github.com/sdwheeler/PowerShell-Docs/pull/3513/commits/1188f78112ba8ccd09dea67f8d4f23d1b7ae6a7e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/3.0/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-3.0&branch=pr-en-us-3513) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/4.0/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-4.0&branch=pr-en-us-3513) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/5.0/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-5.0&branch=pr-en-us-3513) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/5.1/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-5.1&branch=pr-en-us-3513) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Test-Connection.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue1504/reference/6/Microsoft.PowerShell.Management/Test-Connection.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-connection?view=powershell-6&branch=pr-en-us-3513) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901081520350466-3513%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xfwxrL1koj4HbhjTshZMC1Q9exdnyGpEkpTbkY8M4xM%3D&st=2019-01-08T15%3A30%3A10Z&se=2019-02-08T15%3A35%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**